### PR TITLE
Edu benefits confirmation #2994

### PR DESCRIPTION
**Pull request by alexose**
Adds a confirmation page.

The data is placeholder data for now-- Will update when the claims API returns the necessary data.

### CODE REVIEW

File: src/js/edu-benefits/containers/SubmitPage.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+				Muskogee, OK 74402-8888
+      </address>
+		);
+
+		// TODO: replace this with actual data from the API
+    return (
+      <SubmitMessage
+          claimType="Education Benefits"
+          confirmation="8-910fks-76"
+          date="Jul 12, 2016"
```
Comment (jbalboni) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: I though we were getting the date back from the api now?

File: src/js/edu-benefits/containers/SubmitPage.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          address={address}
+          claimedBenefits="Chapter 33"
+          relinquishedBenefits="Chapter 1607"/>
```
Comment (jbalboni) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: Could we pull these from the form data we have? They're both in the form on the benefits selection page.

File: src/js/edu-benefits/containers/SubmitPage.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    );
+  }
+}
+
+function mapStateToProps(state, ownProps) {
+  return {
+    currentLocation: ownProps.location.pathname
```
Comment (jbalboni) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: Do we need this?

File: src/js/rx/containers/Main.jsx [hunk 4]
```diff
@@ -12,15 +12,13 @@ class Main extends React.Component {
...
-      alertBox = (
-        <AlertBox
-            content={this.props.alert.content}
-            isVisible={this.props.alert.visible}
-            onCloseAlert={this.props.closeAlert}
-            status={this.props.alert.status}/>
-      );
-    }
+    alertBox = (
+      <AlertBox
```
Comment (jbalboni) [inline src/js/rx/containers/Main.jsx]: Is it ok for us to use this? There are some css classes prefixed with rx in the code for it.

File: src/js/edu-benefits/containers/SubmitPage.jsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import SubmitMessage from '../components/SubmitMessage';
+
+class SubmitPage extends React.Component {
+  render() {
+		// TODO: shared address component?
+    const address = (
```
Comment (jbalboni) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: I think we'd want to get this from the api eventually, too? I think region depends on some data in the form.

**Review by jbalboni — APPROVED** [review]
This looks good to me. I think there's some more data we could fill in from the form and had some other minor questions.

File: src/js/edu-benefits/containers/SubmitPage.jsx (hunk 2 above)
Comment (alexose) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: We could, although I'd rather see all of this information come back from the server.  If that proves to be tricky, we can definitely pull it from the store.

File: src/js/edu-benefits/containers/SubmitPage.jsx (hunk 3 above)
Comment (alexose) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: Nooope!

File: src/js/rx/containers/Main.jsx (hunk 4 above)
Comment (alexose) [inline src/js/rx/containers/Main.jsx]: It seems like a useful component to have.  I'd rather fix it in `./common` than reinvent the wheel... I think I'll do that

File: src/js/edu-benefits/containers/SubmitPage.jsx (hunk 5 above)
Comment (jbalboni) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: Looks like we'll be getting a `processedAt` property, so we'd probably use that somehow.

**Comment by alexose** [comment]
@U-DON I changed out the rx-alert prefixes on the AlertBox component so that we could use it in edu-benefits.  Hopefully that's all right with you!

I also fixed the problem with SVGs not loading ( #2930 ).  The problem was... weird.  In short, webpack was trying to inline them incorrectly.  I added a module that does it the right way, and it seems to be working now:

image

File: src/js/rx/containers/Main.jsx [hunk 6]
```diff
@@ -12,15 +12,13 @@ class Main extends React.Component {
...
-    if (this.props.alert.visible) {
-      alertBox = (
-        <AlertBox
-            content={this.props.alert.content}
-            isVisible={this.props.alert.visible}
-            onCloseAlert={this.props.closeAlert}
-            status={this.props.alert.status}/>
-      );
-    }
+    alertBox = (
```
Comment (U-DON) [inline src/js/rx/containers/Main.jsx]: Why doesn't this have the conditional anymore? Is its visibility triggered by something else now?

Comment (alexose) [inline src/js/rx/containers/Main.jsx]: Oops, didn't mean to commit that.  Reverted.

**Pull request merged**